osrf/subt
# log detach events, and new performer detector header information

To test:

1. Run simulation: `ign launch cave_circuit.ign worldName:=simple_cave_01 robotName1:=X1 robotConfig1:=X1_SENSOR_CONFIG_1 robotName2:=X2 robotConfig2:=X3_SENSOR_CONFIG_1 marsupial1:=X1:X2`

2. Tail the `events` yaml file: `tail -f /tmp/ign/logs/events.yml`

3. Detach the X2: `rostopic pub /X2/detach std_msgs/Empty "{}"`

4. Teleop the robots through the cave entrance to see the extra header information. You'll need version 2 of https://app.ignitionrobotics.org/OpenRobotics/fuel/models/Cave%20Starting%20Area%20Type%20B

Signed-off-by: Nate Koenig <nate@openrobotics.org>
